app/src/main/java/io/zenandroid/onlinego/views/BoardView.kt

```diff
@@ -165,13 +182,19 @@ class BoardView : View {
         //
         // We need to adapt to resize events by recomputing the border and cell sizes
         //
-        cellSize = w / this.boardSize
+        var usableWidth = w // available width will be adjusted when coordinates are active
+        if (prefs.getBoolean("show_coordinates", false) && !isGameCard) {
+            usableWidth -= (w / this.boardSize.toFloat() * 0.65).roundToInt()
```

Comment:
can you please explain where is this .65 coming from? Unless I misunderstand the code, it does seem that you are reserving 35% of the available space for coordinates, which seems a bit much (?).

Reply:
It enlarges the border by 65% of a cell size without coordinates ( w / boardSize ), meaning less than half a cell space per side for the coordinates. 

as for "why 65% of a cell size", trial and error :) I experimented with combinations of border and text ratios until I was happy with the result on all board sizes:

![screenshot_20190113-204724](https://user-images.githubusercontent.com/12851952/51089957-46537d80-1775-11e9-8dfc-89f329ceb395.png)
![screenshot_20190113-204739](https://user-images.githubusercontent.com/12851952/51089958-46537d80-1775-11e9-8bc4-cbfa32706ea9.png)
![screenshot_20190113-204730](https://user-images.githubusercontent.com/12851952/51089959-46537d80-1775-11e9-84f0-fa11d25308ac.png)

Reply:
Ah, I see, I missed the `-=`, my bad. Just for the sake of me not understanding the code immediately, can we please rewrite it using the kotlin `if` expression?
```
cellSize = if(showCoordinates) {
val usableWidth = w - (w / this.boardSize.toFloat() * 0.65).roundToInt()
usableWidth / boardSize
} else {
w / this.boardSize
}
```
(or something like that)